### ISSUE TITLE
How to change Class label when the model is trained

### ISSUE THREAD

**Issue by hammad2008** [ultralytics/yolov5#8141]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello I hope you are doing well. I have a dataset in which fruit is label with 0 in numeric form. I want to change the class label from 0 to orange How can I change that. The dataset is publicly available. it does not contain labelmap.txt, when i upload it into roboflow website it gives a error 
`
Please add your labelmap text file (if you have one) to translate the numeric class identifiers into human-readable names.
`
The following image is the label batch of validation set.
But i want numeric 0 to change it into orange

![val_batch1_labels](https://user-images.githubusercontent.com/75312856/172532198-270e044f-8e4f-4a97-a93f-0dcdfb9acb4e.jpg)

**Comment by yeldarby** [comment]
In Roboflow you can either drop in [a txt file with a single line that has the class name](https://blog.roboflow.com/label-map/) (during upload time when you get that missing labelmap message) or use [the “remap classes” preprocessing step](https://blog.roboflow.com/introducing-class-management/) when generating a version.

**Comment by glenn-jocher** [comment]
@hammad2008 use PyTorch Hub YOLOv5 models and then `model.names = ['fruit']`
